### PR TITLE
Disabling TCF stub with tcfEnabled

### LINKED ISSUE #24: Option to disable TCF and not load the stub
Some clients doesn't need the TCF stub as they decide not to enable the TCF.

We should add an optional prop `tcfEnabled` which would be `true` by default and could accept boolean values (don't forget to add the type in the typescript interface). When it is set to `false`, we shouldn't load any stub.

### PR DESCRIPTION
**Pull request by jawadst**
This PR introduces a new `embedTCFStub` prop that controls whether the TCF stub should be embedded on the page or not.

Closes #24

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -155,6 +156,15 @@ const DidomiSDK = ({
     return apiKeyProp || apiKey;
   }
 
+  /**
+   * Determine whether the TCF is enabled from the props or from the config
+   */
+  const isTCFEnabled = config && config.app
```
Comment (jawadst) [inline src/index.js:162]: @MaximePlancke I am reading from both `tcfEnabled` and `config.app.vendors.iab.enabled`.

We could also decide that `tcfEnabled` should be set into `config.app.vendors.iab.enabled` and that the prop overrides the config. I think that would make sense here.

**Comment by jawadst** [comment]
@MaximePlancke Can you review?

File: src/index.js (hunk 1 above)
Comment (MaximePlancke) [reply src/index.js:162]: Yes that works for me

**Comment by MaximePlancke** [comment]
@jawadst Can we add the documentation?

**Comment by jawadst** [comment]
@MaximePlancke My bad, added in https://github.com/didomi/react/pull/27/commits/0ab6a0a3157f8bde7d96d5f43fcfabb68123c47e

File: src/index.js (hunk 1 above)
Comment (jawadst) [reply src/index.js]: Done in https://github.com/didomi/react/pull/27/commits/6e91fc3a7499ba64d696777b57221f44ba117ccb

**Review by MaximePlancke — APPROVED** [review]

**Pull request merged**